boms/cloud-lts-bom/pom.xml

```diff
@@ -77,6 +76,7 @@
     <google.cloud.trace.version>1.3.0-sp.1</google.cloud.trace.version>
     <google.cloud.pubsub.version>1.111.0-sp.1</google.cloud.pubsub.version>
     <google.cloud.bigquery.version>1.127.12-sp.1</google.cloud.bigquery.version>
+    <google.cloud.storage.version>1.113.14-sp.1</google.cloud.storage.version>
```

Comment:
Do you want to arrange them alphabetical order? With the alphabetical order, we can easily spot duplicates if any.

Reply:
If that's important, let's write a test.